README.md

```diff
@@ -700,6 +700,32 @@ interface Table extends Parent {
 
 **Table** represents 2d data.
 
+### CustomCodeComponent
+
+```ts
+interface CustomCodeComponent extends Node {
+  type: "custom-code-component"
+  path: string
+  versionRange: string
+  lastModified: string
+  dataIdentifier: string
+  attributes: {
+    [key: string]: string | boolean | undefined
+  }
+  children: CustomCodeComponentChildren[]
+}
+```
+
+**CustomCodeComponent** represents a Custom Code Component block.
```

Comment:
Could this comment be more descriptive, explaining what a custom code component is/why it used by editorial? Something like the comments on the `Recommended` [type](https://github.com/Financial-Times/content-tree/blob/78b125e4cfd55a0c36b9b27230dbc3e82da7b973/README.md#recommended).

Reply:
Thanks @Marlon-Buckley  I have updated [here](https://github.com/Financial-Times/content-tree/pull/52/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R720-R723)